app/models/user.rb

```diff
@@ -3,6 +3,14 @@ class User < ApplicationRecord
   has_many :likes, foreign_key: :author_id
   has_many :comments, foreign_key: :author_id
 
+  validates_associated :posts, :likes, :comments
+
+  validates :name, length: { minimum: 2, maximum: 30 }, allow_blank: false
+  validates :bio, length: { maximum: 500 }, allow_blank: true
+  validates :photo,
```

Comment:
- [ ] kindly remove validation from `bio`.
- [ ]  kindly remove validation from `photo` also otherwise it will create a problem when creating from the console.